Client/Frontend/Browser/LoginsHelper.swift

```diff
@@ -228,7 +228,7 @@ class LoginsHelper: TabHelper {
             }
 
             let json = JSON(jsonObj)
-            let src = "window.__firefox__.logins.inject(\(json.rawString()))"
+            let src = "window.__firefox__.logins.inject(\(json.rawString()!))"
```

Comment:
Also, to answer your question about how this fixes the regression, the string interpolation was resulting in "Optional({...})" which was being interpreted in JavaScript as a call to a non-existent `Optional()` function. But, you're right... if for some reason this returns nil, we may be in trouble. I don't think that can happen, but we should probably add `?? ""` (or something similar) before we uplift this.